lab_1_classify_by_unigrams/main.py

```diff
@@ -64,6 +93,21 @@ def calculate_mse(predicted: list, actual: list) -> float | None:
 
     In case of corrupt input arguments, None is returned
     """
+    """
```

Comment:
does it work for you?

Reply:
Unfortunately, it does not. I tried to fix it, but unsuccessfully